org.videolan.VLC.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
```

Comment:
Why are you making a desktop file, surely upstream installs one and it is probably better including translations and the mimetypes it supports?